README.md

```diff
@@ -1,94 +1,62 @@
-
-NOTE: Team Comtress 2 is no longer compatible with mastercomfig. Please do not use mastercomfig or any other TF2 config.
-
-See [the Valve dev wiki page](https://developer.valvesoftware.com/wiki/Installing_and_Debugging_the_Source_Code) for another explanation of the last two steps.
-
-Other launch options to consider:
-- `sw` to force windowed mode
-- `-w WIDTH -h HEIGHT` to set the resolution
-- `+map MAPNAME` to automatically launch a map on startup
-
-## Legal
-
-Valve, the Valve logo, Steam, the Steam logo, Team Fortress, the Team Fortress logo, Source, the Source logo are trademarks and/or registered trademarks of Valve Corporation in the U.S. and/or other countries.
-
-Team Comtress 2 is not sponsored, endorsed, licensed by, or affiliated with Valve Corporation.
-
-See license for details.
+[![ko-fi](https://img.shields.io/badge/Support%20me%20on-Ko--fi-FF5E5B.svg?logo=ko-fi&style=flat-square)](https://ko-fi.com/mastercoms)
+[![Liberapay](https://img.shields.io/liberapay/receives/mastercoms.svg?logo=liberapay&style=flat-square)](https://liberapay.com/mastercoms/)
+[![Steam donate](https://img.shields.io/badge/Donate%20via-Steam-00adee.svg?style=flat-square&logo=steam)](https://steamcommunity.com/tradeoffer/new/?partner=85845165&token=M9cQHh8N)
+[![Join the Discord chat](https://img.shields.io/badge/Discord-%23comtress--client-7289da.svg?style=flat-square&logo=discord)](https://discord.gg/CuPb2zV)
+
+# Welcome to Team Comtress 2!
+
+### What *is* TC2?
+
+Team Comtress 2 is a fork of an older version of Team Fortress 2, with a community development team fixing bugs, improving performance, and adding quality of life features, with the goal of having those changes pulled upstream by Valve to the modern game.
+
+### What is TC2 *not*?
+* **TC2 is *not* a "pro mod"**
+While much of the work in TC2 will improve the competitive experience, TC2 does not touch gameplay, balance, or competitive rules.
+* **TC2 is *not* a vision for the "good old days" of TF2**
+TC2 will not feature game design changes, gameplay additions, or almost any form of new content. If you're looking for this kind of content, [Team Fortress 2 Classic](https://tf2classic.com/) is more up your alley.
+* **TC2 is *not* the modern game, with new content/updates**
+You're looking for [Creators.TF](https://creators.tf/).
+* **TC2 is *not* an alternative, more stable game experience**
+TC2 is based on the 2017 TF2 source code leak, which contained an in-development version of Jungle Inferno. This build is rife with issues and incomplete features, and ***is not compatible*** with the modern game client.
+
+## How can I help?
+
+1. **Report bugs that exist in *both* TF2 *and* TC2 through the [issues page](https://github.com/mastercomfig/team-comtress-2/issues).**
```

Comment:
They can also report bugs that just happen in TC2

Reply:
Is this wording better?
```
1. **Report bugs that exist in both live TF2 and TC2 (or in just TC2) through the [issues page](https://github.com/mastercomfig/team-comtress-2/issues).**
	* Bugs that are present in modern TF2 but are *not* present in TC2 cannot be fixed in this project.
```